docs/latest/book.json

```diff
@@ -12,7 +12,7 @@
   ],
   "pluginsConfig": {
     "edit-link": {
-      "base": "https://github.com/cornerstonejs/cornerstoneTools/edit/master/docs",
+      "base": "https://github.com/cornerstonejs/cornerstoneTools/edit/master/docs/latest",
```

Comment:
Ah. This makes sense. I guess very few people will want to update docs for previous versions.